components/cockpit.jsx

```diff
@@ -0,0 +1,173 @@
+import styled, {css} from "styled-components";
+import { useState } from "react";
+import { nanoid } from 'nanoid';
+
+export function Cockpit(){
+
+const [commandLine, setCommandLine] = useState([])
+const [count, setCount]= useState(0)
```

Comment:
`count` does not need to be a separate state. It can be fully derived from the `commandLine` array.

```suggestion
const count = commandLine.length;
```

You can then also get rid of all `setCount` calls - it will be up to date immediately.

See this explanation here: https://beta.reactjs.org/learn/choosing-the-state-structure#avoid-redundant-state

Reply:
makes sense, thank you!